pkg/apis/v1alpha5/provisioner_validation.go

```diff
@@ -305,3 +307,35 @@ func ValidateRequirement(requirement v1.NodeSelectorRequirement) error { //nolin
 	}
 	return errs
 }
+
+// Validate validateImageGCHighThresholdPercent
+func (kc *KubeletConfiguration) validateImageGCHighThresholdPercent() (errs *apis.FieldError) {
```

Comment:
Is it possible to build these validations into the CRD itself? https://book.kubebuilder.io/reference/markers/crd-validation.html